### PR TITLE
feat(default-theme): added tip popup on product page

### LINKED ISSUE #394: [FEATURE]Product page- add popup with TIP on mobile
**Describe the solution you'd considered**
On mobile - add popup window with TIP  'Press and hold item to drag them to the cart' on product page as below:
tip

**Acceptance criteria**

- [ ] the popup window appears when user opens product page
- [ ] user can close the popup window after pressing x sign in the corner of the popup

### PR DESCRIPTION
**Pull request by Malwurf**
## Changes

closes #394 

Screenshot from 2020-10-02 14-48-21

### Checklist

- [x] the [list of features](docs/guide/FEATURELIST.md) is updated (if it's a feature and it's relevant)
- [x] I followed [contributing](https://github.com/DivanteLtd/shopware-pwa/blob/master/CONTRIBUTING.md) guidelines

### CODE REVIEW

File: packages/default-theme/components/SwProductDetails.vue [hunk 1]
```diff
@@ -1,5 +1,7 @@
 <template>
   <div class="sw-product-details">
+    <SwTipPopup />
```
Comment (patzick) [inline packages/default-theme/components/SwProductDetails.vue:3]: here should be v-if isMobile and component should be lazy-loaded only in mobile

File: packages/default-theme/components/atoms/SwTipPopup.vue [hunk 2]
```diff
@@ -0,0 +1,94 @@
...
+        size="16px"
+        @click="closePopup"
+      />
+    </div>
+  </transition>
+</template>
+
+<script>
+import { SfIcon } from "@storefront-ui/vue"
+import {
```
Comment (patzick) [inline packages/default-theme/components/atoms/SwTipPopup.vue:22]: we don't need a mobile observer in this component

File: packages/default-theme/components/atoms/SwTipPopup.vue [hunk 3]
```diff
@@ -0,0 +1,94 @@
...
+  },
+  computed: {
+    ...mapMobileObserver(),
+  },
+
+  beforeDestroy() {
+    unMapMobileObserver()
+  },
+  mounted() {
+    if (document.cookie.includes("productPageTip=accepted")) {
```
Comment (patzick) [inline packages/default-theme/components/atoms/SwTipPopup.vue:52]: we have lob for cookies - you can access that via this.$cookies

example
image

**Review by patzick — CHANGES_REQUESTED** [review]
Few change requests from me.

Most important - I believe this PR should be blocked until the functionality described in TIP is implemented, as it would be confusing for users.

File: packages/default-theme/components/SwProductDetails.vue (hunk 1 above)
Comment (Malwurf) [reply packages/default-theme/components/SwProductDetails.vue]: Added isMobile here

File: packages/default-theme/components/atoms/SwTipPopup.vue (hunk 2 above)
Comment (Malwurf) [reply packages/default-theme/components/atoms/SwTipPopup.vue:22]: Removed unnecessary observer

File: packages/default-theme/components/atoms/SwTipPopup.vue (hunk 3 above)
Comment (Malwurf) [reply packages/default-theme/components/atoms/SwTipPopup.vue:52]: Changed it!

**Pull request closed without merging**